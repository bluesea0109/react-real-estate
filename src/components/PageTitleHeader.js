import React from 'react';
import PropTypes from 'prop-types';

import { isMobile } from './utils';
import { Segment } from './Base';

const PageTitleHeader = ({ children, ...rest }) => {
  const style = {
    boxShadow: 'none',
    border: 'none',
    borderBottom: '1px solid lightgrey',
    borderRadius: 0,
  };

  return (
    <Segment style={isMobile() ? { marginTop: '58px', ...style } : { ...style }} {...rest}>
      {children}
    </Segment>
  );
};

PageTitleHeader.propTypes = {
  children: PropTypes.node,
};

export default PageTitleHeader;
