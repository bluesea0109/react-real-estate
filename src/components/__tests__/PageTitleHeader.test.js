import React from 'react';
import renderer from 'react-test-renderer';
import PageTitleHeader from '../PageTitleHeader';
import 'jest-styled-components';

it('renders correctly', () => {
  const tree = renderer.create(<PageTitleHeader />).toJSON();
  expect(tree).toMatchSnapshot();
});
