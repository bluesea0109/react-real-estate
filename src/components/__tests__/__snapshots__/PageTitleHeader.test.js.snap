// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ui segment "
  style={
    Object {
      "border": "none",
      "borderRadius": "4px",
      "boxShadow": "rgba(34, 36, 38, 0) 0px 2px 0px 0px, rgba(34, 36, 38, 0.1) 0px 2px 5px 0px",
      "padding": "0.5rem 1rem",
    }
  }
/>
`;
